data/build_probe_1_data.py

```diff
@@ -0,0 +1,163 @@
+import json
```

Comment:
One high level comment, the names "Probe1" and "Probe 2" are not very expressive.
I know that within the team  we know what it means . But for the public audience visiting the repo it will not be as clear.

My suggestion is either:
1) Chose better names for the probes
2) or Make sure they are documented within this public repo . YOu can add a README.md file with the description of each probe and also describe them in the module docstrings.

Reply:
I made it an issue. I have to rename the metric and evaluation files too so I think I'll do it in one goal on all four probes.